content/docs/reference/design-docs/timetable/index.en.md

```diff
@@ -149,6 +149,8 @@ constraint_distribution: MARECO # or LINEAR
 #      speed on approach as well as short slip distance speed.
 #      This is used for cases were a movable element is placed shortly after the signal
 #      and going beyond the signal would cause major problems.
+#      This is used automatically for any stop before a buffer-stop.
+#      This is also the default use for STDCM stops, as it is the most restrictive.
 schedule:
  - {at: a, stop_for: PT5M, locked: true} # infered arrival to be equal to start_time
```

Comment:
```suggestion
 - {at: a, stop_for: PT5M, locked: true} # inferred arrival to be equal to start_time
```
https://www.merriam-webster.com/dictionary/infered

Reply:
thanks, done (push-forced) in https://github.com/OpenRailAssociation/osrd-website/compare/6bf06f209952d3dd5dd773297873660ae8296e2f..e1bf12960d026baf195c9a0e3d8dd80d9c08cd98